Source/Blazorise.AntDesign/Styles/_switch.scss

```diff
@@ -0,0 +1,37 @@
+﻿@import "variables";
+
+.ant-switch-primary.ant-switch-checked {
```

Comment:
It's better to use scss loop instead of setting each color individualy

```scss
@each $name, $value in $theme-colors {
    $color: $value;
    $color-invert: color-yiq($value);

    &.something-#{$name} {
        background-color: $color;
    }
}
```